src/core/Table/Filters/TextFilter/TextFilter.tsx

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import '@itwin/itwinui-css/css/table.css';
+import { useTheme } from '../../../utils/hooks/useTheme';
+import { TableFilterProps } from '../BaseFilter/BaseFilter';
+import { Input } from '../../../Input';
+import { FilterButtonBar } from '../FilterButtonBar';
+
+export const TextFilter = <T extends Record<string, unknown>>(
+  props: TableFilterProps<T>,
+) => {
+  const { column, setFilter, clearFilter } = props;
+
+  useTheme();
+
+  const [text, setText] = React.useState(column.filterValue);
+
+  return (
+    <>
+      <Input value={text} onChange={(e) => setText(e.target.value)} setFocus />
+      <FilterButtonBar
```

Comment:
I wanted to integrate this into `BaseFilter` but this was that prevented me:
![image](https://user-images.githubusercontent.com/36186912/117451679-7b3dfc80-af4b-11eb-9e71-df4aaa7aa61c.png)
But later I thought that maybe this will make `BaseFilter` more reusable.

Reply:
I dont understand... Why BaseFilter is there, if it is not reused by this?